ethpm/tools/builder.py

```diff
@@ -524,14 +527,35 @@ def _to_disk(
     else:
         disk_manifest_name = manifest["version"] + ".json"
 
-    if prettify:
-        contents = json.dumps(manifest, sort_keys=True, indent=4)
-    else:
-        contents = json.dumps(manifest, sort_keys=True, separators=(",", ":"))
+    contents = format_manifest(manifest, prettify)
 
     if (cwd / disk_manifest_name).is_file():
         raise ManifestBuildingError(
             "Manifest: {0} already exists in cwd: {1}".format(disk_manifest_name, cwd)
         )
     (cwd / disk_manifest_name).write_text(contents)
     return manifest
+
+
+def to_ipfs(
+    ipfs_backend: BaseIPFSBackend, prettify: Optional[bool] = False
+) -> List[Dict[str, str]]:
+    """
+    Returns the IPFS pin data after pinning the manifest to the provided IPFS Backend.
+
+    `to_ipfs()` Should *always* be the last argument in a builder, as it will return the pin
+    data and not the manifest.
+    """
+    return _to_ipfs(ipfs_backend, prettify)
+
+
+@cytoolz.curry
+def _to_ipfs(
```

Comment:
I don't think you need the two functions.  That pattern is only really necessary when you need to do `*args`.  I think you can just reverse the argument order here so that `manifest` is the first argument and then require the others to be keyword

```python
def to_ipfs(manifest, *, ipfs_backend, prettify=False):
    ...
```

This will have the elegant effect of requiring both `prettify` and `ipfs_backend` to be passed as keywoard arguments.